### PR TITLE
Add Pocket Best of 2022 home promo (Fixes #12406)

### LINKED ISSUE #12406: Homepage update for pocket best of
## Description

We will be updating the homepage to reflect Pocket's 'best of' campaign. 

The design will be mostly shared with the designs we are using for our December WNP.

[Deck with designs & copy](https://docs.google.com/presentation/d/1YWjdaK_A6y_CNA-3d0XULHzsmKG0Nc7pZXiNqVLBQLs/edit?usp=sharing)

---

#### Success Criteria
- [x] proper UTM params in place for tracking.

### PR DESCRIPTION
**Pull request by alexgibson**
## One-line summary

- Add Pocket Best of 2022 home page promo for English and German.
- Removes older Pocket 10 years promo.

Promo is behind the switch `SWITCH_HOMEPAGE_POCKET_BEST_2022_PROMO`

## Issue / Bugzilla link

#12406

## Testing

- http://localhost:8000/en-US/
- http://localhost:8000/de/

### CODE REVIEW

**Comment by alexgibson** [comment]
This is ready for code review, but still waiting on stakeholder approval.

**Comment by alexgibson** [comment]
This has been approved by stakeholders so if fine to merge once reviewed.

**Comment by alexgibson** [comment]
Ok, ready

File: bedrock/mozorg/templates/mozorg/home/includes/pocket-best-2022-promo.html [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  {% set promo_desc = 'In a year of ups and downs, these articles helped us make sense of it all. See what made the list.' %}
+  {% set promo_cta = 'See the winners' %}
+  {% set promo_url = 'https://getpocket.com/collections/pocket-best-articles-2022' %}
+{% endif %}
+
+{% call split(
+  media_include='mozorg/home/includes/pocket-best-2022-bg.svg',
+  block_class='pocket-best-2022-promo mzp-l-split-body-wide mzp-l-split-center-on-sm-md mzp-t-content-xl',
+  media_class='mzp-l-split-h-end mzp-l-split-v-center',
+  image=picture(
```
Comment (maureenlholland) [inline bedrock/mozorg/templates/mozorg/home/includes/pocket-best-2022-promo.html:25]: 👏 👏 great use of new image param and picture art direction

File: media/css/mozorg/home/home-pocket-best-2022-promo.scss [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+        & > :last-child {
+            margin-bottom: $spacing-xl;
+        }
+    }
+
+    .mzp-c-split-media-asset {
+        height: auto;
+        max-width: none;
+        vertical-align: middle;
```
Comment (maureenlholland) [inline media/css/mozorg/home/home-pocket-best-2022-promo.scss:42]: nice use of `vertical-align`!

File: media/css/mozorg/home/home-pocket-best-2022-promo.scss [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+@import '~@mozilla-protocol/core/protocol/css/components/logos/wordmark';
+@import '~@mozilla-protocol/core/protocol/css/components/logos/wordmark-product-pocket';
+@import '~@mozilla-protocol/core/protocol/css/includes/lib';
+
+.pocket-best-2022-promo {
+    background-color: #dceaff;
+    padding: 0;
+
+    .pocket-best-2022-promo-title {
+        @include text-display-sm;
```
Comment (maureenlholland) [inline media/css/mozorg/home/home-pocket-best-2022-promo.scss:17]: **non-blocking, question**: I don't think I've seen the `display` mixins in use, are they meant to be deprecated? https://github.com/mozilla/protocol/blob/7cbfb52acef30c67f324202db187c09992f071b1/assets/sass/protocol/includes/mixins/_typography.scss#L151

File: bedrock/mozorg/templates/mozorg/home/includes/pocket-10-years-promo.html [hunk 4]
```diff
@@ -1,37 +0,0 @@
...
-  License, v. 2.0. If a copy of the MPL was not distributed with this
-  file, You can obtain one at https://mozilla.org/MPL/2.0/.
-#}
-
-{% from "macros-protocol.html" import split with context %}
-
-{% call split(
-  image_url='img/home/2021/pocket-10-years-700.jpg',
-  image_srcset={
-    'img/home/2021/pocket-10-years-700.jpg': '700w',
```
Comment (maureenlholland) [inline bedrock/mozorg/templates/mozorg/home/includes/pocket-10-years-promo.html:12]: **chore**: delete 10 year images no longer in use

**Review by maureenlholland — APPROVED** [review]
Looks great! 🎉  UTM in place, switch works as expected ✅ (best of links currently 404, but I assume they've confirmed those paths, so all will be good to go once the pages exist and the switch is flipped)

r+wc, bit of media clean-up and one non-blocking question

File: media/css/mozorg/home/home-pocket-best-2022-promo.scss (hunk 3 above)
Comment (alexgibson) [reply media/css/mozorg/home/home-pocket-best-2022-promo.scss:17]: Ah, so they are  - will update to `title`!

**Comment by alexgibson** [comment]
@maureenlholland thanks for always being so proactive on reviews (and giving great feedback), updated!

**Review by maureenlholland — APPROVED** [review]
r++ 💯

**Pull request merged**